src/components/sidekick/index.test.tsx

```diff
@@ -6,10 +6,21 @@ import { IfAuthenticated } from '../authentication/if-authenticated';
 import { MessengerList } from '../messenger/list';
 
 describe('Sidekick', () => {
+  beforeAll(() => {
+    global.localStorage = {
+      setItem: jest.fn(),
+      getItem: () => null,
+      removeItem: () => {},
+      length: 0,
+      clear: () => {},
+      key: (_) => '',
+    };
```

Comment:
I suspect you'll have to mock the local store for all tests if you move the lookup into the store. I think you should be able to move this into the `src/setupTests.ts` file.